daemon/algod/api/algod.oas2.json

```diff
@@ -5544,6 +5545,10 @@
             "description": "OnlineMoney",
             "type": "integer"
           },
+          "online-circulation": {
+            "description": "OnlineCirculation as used by consensus, excluding expired accounts",
```

Comment:
maybe add some clarification about the value: like not a value at the moment but at -320 rounds back as consensus defines? Or it is too much?